src/Razor/test/Microsoft.VisualStudio.Razor.IntegrationTests/CodeFoldingTests.cs

```diff
@@ -121,7 +121,7 @@ static CollapsibleBlock ConvertToLineNumbers(Span span, ITextView textView)
             }
         }
 
-        [IdeFact(Skip = "https://github.com/dotnet/razor-tooling/issues/6966")]
+        [IdeFact]
```

Comment:
Though we have not fixed all the issues here my experiments lead me to believe that the ITextBuffer change is sufficient to get us to roughly as reliable as the other tests. I'll run the tests a couple times to see if that bears out on the CI.